README.md

```diff
@@ -2,11 +2,12 @@
 
 [![Build Status](https://travis-ci.org/oicr-gsi/pinery-reports.svg)](https://travis-ci.org/oicr-gsi/pinery-reports)
 
-A Java app for generating regular reports based on Pinery data
+A Java app for generating regular reports based on [Pinery](https://github.com/oicr-gsi/pinery) data
 
 ## Build
 
 ```
+git checkout develop
```

Comment:
Shouldn't be a build step since you may want to build a different branch or tag. You could note that `develop` is the main development branch and that tags exist for releases.

I'd rather the main branch were called `master` as is standard. `develop` branches are generally used in the weird situation where you want there to exist a `master` branch that points to a release despite tags being better suited for such a thing. Just my 2 cents :grin:

Reply:
I'm gonna merge this for now, but let's have a talk about this next week!